dot/utils_test.go

```diff
@@ -145,7 +144,7 @@ func TestNewTestConfig(t *testing.T) {
 					Modules: []string{"system", "author", "chain", "state", "rpc", "grandpa", "offchain",
 						"childstate", "syncstate", "payment"},
 					WSPort:           8546,
-					WS:               false,
+					WS:               true,
```

Comment:
nit why this change for `TestNewTestConfig`

Reply:
to match with the westend config